frontend/pages/hubs/[hubUrl]/browse.tsx

```diff
@@ -324,98 +303,3 @@ export default function Hub({
     </>
   );
 }
-
-const FabShareButton = ({ locale, hubAmbassador, isCustomHub, hubUrl }) => {
-  const fabClass = shareProjectFabStyle({ isCustomHub: isCustomHub });
-  const queryString = hubUrl ? `?hub=${hubUrl}` : "";
-  return (
-    <Fab
-      className={fabClass.fabShareProject}
-      size="medium"
-      color="primary"
-      href={`${getLocalePrefix(locale)}/share${queryString}`}
-      sx={{ bottom: (theme) => (hubAmbassador ? theme.spacing(11.5) : theme.spacing(5)) }}
-      // onClick={}
-    >
-      <AddIcon />
-    </Fab>
-  );
-};
```

Comment:
moved out this part of the code to frontend/src/components/hub/FabShareButton.tsx